File: app/src/main/java/in/mvpstarter/sample/util/rx/scheduler/ComputationMainScheduler.java
package in.mvpstarter.sample.util.rx.scheduler;

import io.reactivex.android.schedulers.AndroidSchedulers;
import io.reactivex.schedulers.Schedulers;

/**
 * Created by lam on 2/6/17.
 */

public class ComputationMainScheduler<T> extends BaseScheduler<T> {

    protected ComputationMainScheduler() {
        super(Schedulers.computation(), AndroidSchedulers.mainThread());
    }
}

File: app/src/main/java/io/mvpstarter/sample/util/rx/scheduler/ComputationMainScheduler.kt
package io.mvpstarter.sample.util.rx.scheduler

import io.reactivex.android.schedulers.AndroidSchedulers
import io.reactivex.schedulers.Schedulers

/**
 * Created by lam on 2/6/17.
 */

class ComputationMainScheduler<T> private constructor() : BaseScheduler<T>(Schedulers.computation(), AndroidSchedulers.mainThread())
